jenkins/node-image/Makefile

```diff
@@ -14,15 +14,38 @@
 
 IMG = gcr.io/k8s-testimages/kubekins-node
 TAG = $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)
+VERSIONS := $(wildcard ./Dockerfile-*)
```

Comment:
wildcard, eval, subst, foreach... I'm a bit worried about the complexity of this Makefile. Especially since we'll only have at most three different versions.

IMO it seems simpler if we just list out the three versions we use at any given time.

However happy to try this out if the rest of the team likes it.